.gitignore

```diff
@@ -128,9 +128,8 @@ server/webapp/WEB-INF/rails.new/log/
 server/webapp/WEB-INF/rails.new/logs/
 server/webapp/WEB-INF/rails.new/node_modules/
 server/webapp/WEB-INF/rails.new/public/assets/
-server/webapp/WEB-INF/rails.new/vendor/bundle/jruby/1.9/doc/
-server/webapp/WEB-INF/rails.new/vendor/bundle/jruby/1.9/extensions/
 server/webapp/WEB-INF/rails.new/webpack/gen/
+server/webapp/WEB-INF/rails.new/vendor/
```

Comment:
As far as I know, you can't gitignore vendor. Some parts are not from the Gemfile. For instance, [see this](https://github.com/gocd/gocd/blob/03f39b7f21154fd41c3fc1f5770edb7540a19d9a/server/webapp/WEB-INF/rails.new/Gemfile#L19).